src/plugins/cosmos/components/modals/GetStarted/views/GetStarted.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, ModalCloseButton, VStack } from '@chakra-ui/react'
+import { AnimatePresence } from 'framer-motion'
+import { DefiModalHeader } from 'plugins/cosmos/components/DefiModalHeader/DefiModalHeader'
+import { useHistory, useLocation } from 'react-router-dom'
+import osmosis from 'assets/osmosis.svg'
+import { Text } from 'components/Text'
+
+type GetStartedProps = {
+  assetId: string
+}
+
+// TODO: Abstract me in a service when I start to get too big
+const ASSET_ID_TO_MAX_APR = {
+  'cosmoshub-4/slip44:118': '12'
+}
```

Comment:
This feels a bit strange, I see where you're going, though it's maybe a premature abstraction.

Reply:
Definitely is, but as discussed in the original PR this was merged in, we're going to use a selector/service for this later on so it won't stay here for long.
Do we want to just hardcode the APR for now as `const maxApr = '12'`?

Reply:
I would, though up to you 👌